SECP-service/src/main/java/com/visucius/secp/Controllers/User/UserErrorMessage.java

```diff
@@ -33,5 +34,11 @@
     String DEVICE_GET_FAIL_NO_DEVICE_EXISTS = "No device exists for user.";
     String DEVICE_GET_FAIL_NO_PUBLIC_KEY_EXISTS = "No public key exist for all user's devices.";
 
-
+    //UserProfile error messages
+    String DISPLAY_NAME_FAILED = "Failed to set the display name.";
+    String DISPLAY_NAME_FAILED_NO_DISPLAY_NAME = "Failed to set the display name. No display name chosen.";
+    String DISPLAY_NAME_FAILED_USER_NOT_FOUND = "Failed to set the display name. User not found.";
+    String AVATAR_URL_FAILED = "Failed to set the Avatar URL.";
```

Comment:
I think this error message can be removed. It does not tell us exactly why it fails. You can just use the AVATAR_URL_FAILED_NO_AVATAR_URL. Same comment goes to DISPLAY_NAME_FAILED.